### DISCUSSION TITLE
Can I create a Collection based on filepath "dynamically"?

### DISCUSSION THREAD

**Discussion by DaydreamHero** [meisnate12/Plex-Meta-Manager#1727] [Q&A] [answered]
Within movies I am using the methodology found at #1640 for my ESPN 30 For 30 films and it works great (takes a bit, but who cares, it does what I need it to do). I have my Stand Up comedy library setup with each comedian in their own folder and within Plex have Smart Collections to look at Actors/Titles so that all Mitch Hedberg files are under the Mitch Hedberg collection, but I was hoping to hand this task over to PMM. Is there a way to do this dynamically so that if I add a new comedian, it would create a new collection based on the root folder name? [Stand Up is a top-level folder similar to Movies, Television, etc.] I am good with re-orging the subfolders/files of the artist's folder if it makes this work. Below is an example of what the folder structure looks like:

> Stand Up
> -Mitch Hedberg
> --Just For Laughs Montreal (2004)
> ---Just For Laughs Montreal (2004).mkv
> --Mitch Hedberg on Conan O'Brien (2003).mp4

**Comment by chazlarson** [comment, marked as answer]
There is not presently a way to dynamically look at the filesystem, since PMM generally does not have access to the backing file system.

You could probably do this with the dynamic actor collections, since for the most part standup shows have a single cast member.

**Reply by DaydreamHero** [reply]
I had thought about that, but some of the stand ups are tv shows some are "movies" so for about half I am adding in the front-end metadata.  I know I can go into the Plex database and change the actors, etc. but that is more complex (with far greater downside) than what I'm already doing. I didn't think it was possible, but figured it was worth a shot. Thanks for the prompt response.
